tmt/utils/hints.py

```diff
@@ -177,3 +179,24 @@ def print_hints(*ids: str, ignore_missing: bool = False, logger: tmt.log.Logger)
 
     for hint in hints:
         logger.info('hint', hint.render(logger), color='blue')
+
+
+def check_for_message(patterns: list[re.Pattern[str]], outputs: list[Optional[str]]) -> bool:
```

Comment:
`outputs` could be `outputs: *Optional[str]`, still iterable, no need to construct a list.

Reply:
This seems to be invalid syntax. I guess you meant `*outputs: Optional[str]` instead?